utils/config/daos_server.yml

```diff
@@ -256,11 +256,21 @@
 #firmware_helper_log_file: /tmp/daos_firmware_helper.log
 #
 #
-## Enable HTTP endpoint for remote telemetry collection.
+## Enable Telemetry HTTP/HTTPS endpoint for remote telemetry collection.
 #
-## default endpoint state: disabled
-## default endpoint port: 9191
-#telemetry_port: 9191
+#telemetry_config:
+#  # In order to disable telemetry security, uncomment and set allow_insecure to false
+#  allow_insecure: true
```

Comment:
same as dmg, will try that tool and remove.

Reply:
Code was modified to use the system certificate. But to be aligned with Transport config option I think it's better we can keep it. This will also gives user a chance to use specific security level with certificates.